From: srinivas pulukuru <srinivas.pulukuru@ti.com>
Date: Wed, 23 Mar 2011 19:13:32 +0000 (-0500)
Subject: OMAP4 DSS HDMI: align video timings with CEA-861-E
X-Git-Tag: android-2.6.35-2.3-omap4.13.1~38
X-Git-Url: http://git.omapzoom.org/?p=kernel%2Fomap.git;a=commitdiff_plain;h=4f6c6fbf3d9729fa049c4fa8771cc2c50d533cc9

OMAP4 DSS HDMI: align video timings with CEA-861-E

The pixel clock was a mix of of 59.94 Hz and 60 Hz.
Pixel clock is rounded in this case

Change-Id: If2614cdbb73b5729a5079492e2a05f7132098dc8
Signed-off-by: Fabrice Olivero <f-olivero@ti.com>
Signed-off-by: Srinivas Pulukuru <srinivas.pulukuru@ti.com>
---

diff --git a/drivers/video/omap2/dss/hdmi.c b/drivers/video/omap2/dss/hdmi.c
index 4a70d23..49d75ca 100644
--- a/drivers/video/omap2/dss/hdmi.c
+++ b/drivers/video/omap2/dss/hdmi.c
@@ -185,9 +185,9 @@ static const struct omap_video_timings all_timings_direct[] = {
 	{640, 480, 25200, 96, 16, 48, 2, 10, 33},
 	{1280, 720, 74250, 40, 440, 220, 5, 5, 20},
 	{1280, 720, 74250, 40, 110, 220, 5, 5, 20},
-	{720, 480, 27027, 62, 16, 60, 6, 9, 30},
+	{720, 480, 27000, 62, 16, 60, 6, 9, 30},
 	{2880, 576, 108000, 256, 48, 272, 5, 5, 39},
-	{1440, 240, 27027, 124, 38, 114, 3, 4, 15},
+	{1440, 240, 27000, 124, 38, 114, 3, 4, 15},
 	{1440, 288, 27000, 126, 24, 138, 3, 2, 19},
 	{1920, 540, 74250, 44, 528, 148, 5, 2, 15},
 	{1920, 540, 74250, 44, 88, 148, 5, 2, 15},
@@ -195,7 +195,7 @@ static const struct omap_video_timings all_timings_direct[] = {
 	{720, 576, 27000, 64, 12, 68, 5, 5, 39},
 	{1440, 576, 54000, 128, 24, 136, 5, 5, 39},
 	{1920, 1080, 148500, 44, 528, 148, 5, 4, 36},
-	{2880, 480, 108108, 248, 64, 240, 6, 9, 30},
+	{2880, 480, 108000, 248, 64, 240, 6, 9, 30},
 	{1920, 1080, 74250, 44, 638, 148, 5, 4, 36},
 	/* Vesa frome here */
 	{640, 480, 25175, 96, 16, 48, 2, 10, 33},